From 4b6dfcb28a37aacf0ff114cd061c7507cfb0f790 Mon Sep 17 00:00:00 2001
From: Alexander Dymo <adymo@pluron.com>
Date: Fri, 12 Jun 2009 12:53:15 +0300
Subject: [PATCH] Print again "<Model> Create" into the log (instead of "SQL")
  on inserting values in postgres >= 8.2.
 "name" parameter wasn't passed to the select_value function.

---
 .../connection_adapters/postgresql_adapter.rb      |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb b/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb
index 8440223..d5f2ed3 100644
--- a/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb
+++ b/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb
@@ -475,7 +475,7 @@ module ActiveRecord
         if supports_insert_with_returning?
           pk, sequence_name = *pk_and_sequence_for(table) unless pk
           if pk
-            id = select_value("#{sql} RETURNING #{quote_column_name(pk)}")
+            id = select_value("#{sql} RETURNING #{quote_column_name(pk)}", name)
             clear_query_cache
             return id
           end
-- 
1.6.0.2

